packages/react-i18n/src/hooks/useTranslate.js

```diff
@@ -4,5 +4,5 @@ import { Context } from '../context/i18n.context';
 export const useTranslate = () => {
   const t = useContext(Context);
 
-  return (key, { data, number, general, renderers } = {}) => t(key, data, number, general, renderers);
```

Comment:
I don't understand if the `number` parameter should really be removed. As I understand the issue, it's only the check by the lint plugin that have to be removed. @flepretre do you confirm?

Reply:
Reverted these two files!

What I don't know about this US is why are we using 1.6.1 whereas there is a 1.9.2 available and even now 2.0.0
Is there something wrong with the last releases?
@flepretre